x-pack/platform/plugins/shared/streams/server/routes/content/route.ts

```diff
@@ -42,17 +45,19 @@ const exportContentRoute = createServerRoute({
 
     await streamsClient.ensureStream(params.path.name);
 
-    const dashboards = await assetClient
-      .getAssets({ entityId: params.path.name, entityType: 'stream' })
-      .then((assets) => assets.filter(({ assetType }) => assetType === 'dashboard'));
+    function isDashboard(asset: Asset): asset is DashboardAsset {
+      return asset[ASSET_TYPE] === 'dashboard';
+    }
+
+    const dashboards = (await assetClient.getAssets(params.path.name)).filter(isDashboard);
```

Comment:
should this support queries as well?

Reply:
Happy to do this in a follow-up PR, I guess it's a wider story: export the queries and import them

Reply:
https://github.com/elastic/streams-program/issues/227